### PR TITLE
fix(647): Moved LoadBalancerEditer component to different path

### LINKED ISSUE #647: Move (Expression,DataFormat,LoadBalancer}Editor components to components/Form
### Please describe the task that needs to be done

cleanup suggestion - https://github.com/KaotoIO/kaoto-next/pull/630#pullrequestreview-1820218254

### PR DESCRIPTION
**Pull request by shivamG640**
Fixes #647

### CODE REVIEW

File: packages/ui/src/components/Visualization/Canvas/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export * from './Canvas';
 export * from './CanvasSideBar';
-export * from './DataFormatEditor';
-export * from './StepExpressionEditor';
+export * from '../../Form/dataFormat/DataFormatEditor';
+export * from '../../Form/stepExpression/StepExpressionEditor';
```
Comment (lordrip) [inline packages/ui/src/components/Visualization/Canvas/index.ts:4]: mmm, this is interesting, I think we should remove these exports from this file and export them from the `Form` folder instead, let me check

Comment (lordrip) [reply packages/ui/src/components/Visualization/Canvas/index.ts:4]: Seems this is not used, so we can delete these exports.

Comment (shivamG640) [reply packages/ui/src/components/Visualization/Canvas/index.ts]: I tried removing them but found out that they are getting used [here](https://github.com/KaotoIO/kaoto-next/tree/main/packages/ui-tests/stories/metadataEditor). So I moved those exports to the `Form` folder. @lordrip

Comment (lordrip) [reply packages/ui/src/components/Visualization/Canvas/index.ts]: Good catch 👌, that helps

**Review by lordrip — APPROVED** [review]

**Pull request merged**